app/src/main/java/ru/practicum/android/diploma/convertor/DbConvertor.kt

```diff
@@ -0,0 +1,23 @@
+package ru.practicum.android.diploma.convertor
+
+import ru.practicum.android.diploma.favorite.data.db.VacancyEntity
+import ru.practicum.android.diploma.search.domain.model.Vacancy
+
+class DbConvertor {
+
+    fun map(vacancy: Vacancy): VacancyEntity {
+        return VacancyEntity(
+            vacancyId = vacancy.vacancyId,
+            vacancyTitle = vacancy.vacancyTitle,
+            vacancyDescription = vacancy.vacancyDescription
+        )
+    }
```

Comment:
Подобные маперы можно писать лаконичнее
```suggestion
    fun map(vacancy: Vacancy): VacancyEntity = VacancyEntity(
            vacancyId = vacancy.vacancyId,
            vacancyTitle = vacancy.vacancyTitle,
            vacancyDescription = vacancy.vacancyDescription
        )
```

Reply:
Решено.
@yulia1Volkova